src/sonic_ax_impl/mibs/ietf/rfc1213.py

```diff
@@ -302,6 +336,31 @@ def _get_counter(self, oid, table_name):
             mibs.logger.warning("SyncD 'COUNTERS_DB' missing attribute '{}'.".format(e))
             return None
 
+    def aggregate_counters(self):
+        """
+        For ports with l3 router interfaces l3 drops may be counted separately (RIF counters)
+        add l3 drops to l2 drop counters cache according to mapping
+
+        For l3vlan map l3 counters to l2 counters
+        """
+        for rif_sai_id, port_sai_id in self.rif_port_map.items():
+            if port_sai_id in self.if_id_map:
+                port_idx = mibs.get_index_from_str(self.if_id_map[port_sai_id])
+                for port_counter_name, rif_counter_name in mibs.RIF_DROPS_AGGR_MAP.items():
+                    self.if_counters[port_idx][port_counter_name] = \
```

Comment:
>self.if_counters [](start = 20, length = 16)

I see you change `self.if_counters[][]` type from `str` to `int`.
Although this is working in python, however we are moving the library to sonic-swss-common SWIG wrapper, and this will make SWIG confused.
Please make type fixed.